hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/records/impl/pb/RouterStoreTokenPBImpl.java

```diff
@@ -164,6 +170,40 @@ public void setRenewDate(Long renewDate) {
     this.builder.setRenewDate(renewDate);
   }
 
+  @Override
+  public byte[] toByteArray() throws IOException {
+    return builder.build().toByteArray();
+  }
+
+  @Override
+  public void readFields(DataInput in) throws IOException {
+    builder.mergeFrom((DataInputStream) in);
+  }
+
+  @Override
+  public String getTokenInfo() {
+    RouterStoreTokenProtoOrBuilder p = viaProto ? proto : builder;
+    if (this.tokenInfo != null) {
+      return this.tokenInfo;
+    }
+    if (!p.hasTokenInfo()) {
+      return null;
+    }
+    this.tokenInfo = p.getTokenInfo();
+    return this.tokenInfo;
+  }
+
+  @Override
+  protected void setTokenInfo(String tokenInfo) {
+    maybeInitBuilder();
+    if(tokenInfo == null) {
```

Comment:
Space

Reply:
BTW, how is checkstyle not catching this?

Reply:
i will fix it.